tachyon/c/examples/prime_field.cc

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+
+#include "tachyon/c/math/elliptic_curves/bn/bn254/fr.h"
+
+int main() {
+  // Initialize prime field elements
+  tachyon_bn254_fr a = tachyon_bn254_fr_random();
+  tachyon_bn254_fr b = tachyon_bn254_fr_random();
+
+  // Display the random elements
+  printf("Random prime field elements a and b have been generated.\n");
+
+  // Addition
+  tachyon_bn254_fr sum = tachyon_bn254_fr_add(&a, &b);
```

Comment:
will any of these results be displayed or used? (aka `sum` or `difference`)

Reply:
We don't have ToString() for c api at the moment, so it would be difficult to display those results.